core/storage/app_feedback_report/gae_models.py

```diff
@@ -351,59 +424,66 @@ class AppFeedbackReportTicketModel(base_models.BaseModel):
 
     # A name for the ticket given by the maintainer, limited to 100 characters.
     ticket_name = datastore_services.StringProperty(required=True, indexed=True)
+    # The platform that the reports in this ticket pertain to.
+    platform = datastore_services.StringProperty(
+        required=True, indexed=True,
+        choices=PLATFORM_CHOICES)
     # The Github repository that has the associated issue for this ticket. The
     # possible values correspond to GITHUB_REPO_CHOICES.
```

Comment:
Please explain what a value of None here means.

Reply:
Done